core/src/main/csharp/ch/cyberduck/core/CollectionObservables.cs

```diff
@@ -0,0 +1,179 @@
+﻿using DynamicData;
+using System;
+using System.Collections.Generic;
+using System.Reactive;
+
+#nullable enable
+
+namespace ch.cyberduck.core;
+
+public static class CollectionObservables
+{
+    public delegate TResult ObservableTransform<TSource, TResult>(TSource obj, TResult? cached);
+
+    public static IObservable<IChangeSet<TResult>> ToObservableChangeSet<TSource, TResult>(
```

Comment:
Create Observable from ch.cyberduck.core.Collection
Basically allows watching for collectionX-notifications